google-cloud-dialogflow/pom.xml

```diff
@@ -73,7 +73,16 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
-
+    <dependency>
+      <groupId>com.google.http-client</groupId>
+      <artifactId>google-http-client</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
This isn't used directly, so it's failing the dependency check. To fix the version convergence, we should update the version of google-cloud-core in the parent pom which will pull in the correct http-client version.

Reply:
Done.